time-calculator/time_calculator.py

```diff
@@ -0,0 +1,67 @@
+
+def add_time(startTime, stopTime, dayOfWeek=None):
+
+    # Start Time
+    [startTime, start_am_pm] = startTime.split()
+    [startHours, startMinutes] = startTime.split(':')
+
+    # End Time
+    [stopHours, stopMinutes] = stopTime.split(":")
+
+    stopMinutes = int(stopMinutes) + (60 * int(stopHours))
```

Comment:
We shouldn't have to use `int` while using these variables i.. these should be already in integers because when we just read the name of the variable then we get the idea that it is integer